### ISSUE TITLE
Is there are doc that diffs the 20181012 and 20180412 pymisp.pdf versions?

### ISSUE THREAD

**Issue by github-germ** [MISP/PyMISP#287]
Thanks for all your hard work on MISP/PyMISP 2.4.96. Looking real good!!

Can you suggest a quick-and-easy way to see what's new/changed in the new pymisp.pdf API doc? (https://media.readthedocs.org/pdf/pymisp/latest/pymisp.pdf)

I may want to revise some prior PyMISP app code I have to incorporate better ways to use the API.

Thanks again -- great job!!

**Comment by Rafiot** [comment]
The changelog is a good starting point. And you can also look at the tutorials, the way stuff is implemented is the recommended approach.

**Comment by github-germ** [comment]
Thx for your reply.

I am asking as part of my learning curve on how best to keep current on great progress and evolution of MISP/PyMISP... (this is mostly process and docs related). Appreciate your help!!

Two small examples, I believe `test_connection` is new to the API. I don't think I saw that mentioned in the changelog (https://github.com/MISP/PyMISP/blob/master/CHANGELOG.txt). When you refer to the  _recommended approach_, does this mean the examples on https://github.com/MISP/PyMISP/tree/master/examples ? If so, I think only _feed-generator-from-redis_ (https://github.com/MISP/PyMISP/tree/master/examples/feed-generator-from-redis) has changed in the last 10 days.

Additionally, I saw in https://github.com/MISP/MISP/issues/2465#issuecomment-431656910 paging was now implemented in the API. Other reviewing issues where how would I have known about this new really nice feature?

Lastly, if one has an open issue that was fixed or somehow addressed in a given release (commit), and there's no new posting to those issues stating that, is it the creators task to re-test to see if the issue is resolved, or is there another method of closure?
